modules/migration-adding-cluster-to-cam.adoc

```diff
@@ -37,16 +37,17 @@ eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiw
 . Click *Add cluster*.
 . Fill in the following fields:
 
-* *Cluster name*: May contain lower-case letters (`a-z`) and numbers (`0-9`). Must not contain spaces or international characters.
-* *Url*: URL of the cluster's API server, for example, `\https://<master1.example.com>:8443`.
-* *Service account token*: String that you obtained from the source cluster.
-* *Exposed route to image registry*: Optional. You can specify a route to the image registry of your source cluster to enable direct migration for images, for example, `docker-registry-default.apps.cluster.com`.
+* *Cluster name*: The cluster name may contain lower-case letters (`a-z`) and numbers (`0-9`). It must not contain spaces or international characters.
```

Comment:
```suggestion
* *Cluster name*: The cluster name can contain lower-case letters (`a-z`) and numbers (`0-9`). It must not contain spaces or international characters.
```

> ! may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.